shell/pages/c/_cluster/explorer/index.vue

```diff
@@ -400,6 +407,21 @@ export default {
     // Events/Alerts tab changed
     tabChange(neu) {
       this.selectedTab = neu?.selectedName;
+    },
+
+    async goToHarvesterCluster() {
+      try {
```

Comment:
Could this be a method on the provisioning cluster type? This would be easier to maintain than having it defined in 3 places as it is now

Reply:
Done